extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialogModel.ts

```diff
@@ -61,6 +82,15 @@ export class MigrationCutoverDialogModel {
 					this._migration.subscription,
 					this.migrationStatus
 				);
+				sendSqlMigrationActionEvent(
+					TelemetryViews.MigrationCutoverDialog,
+					TelemetryAction.CancelMigration,
+					{
+						'sessionId': this._migration.sessionId!,
+						'cutoverStartTime': new Date().toString()
```

Comment:
This doesn't seem like the right event for sending cutoverStartTime - how do you know when the error actually occurred? Normally you'd capture the start time before starting the cutover and then include it in any relevant events afterwards.

Reply:
The cutover start api to is asynchronous and returns a response immediately. However, I agree calculating the start time before calling the api will be more accurate.